lib/base64.c

```diff
@@ -187,6 +179,7 @@ static CURLcode base64_encode(const char *table64,
   char *output;
   char *base64data;
   const char *indata = inputbuff;
+  const char *padstr = table64 + 64;
```

Comment:
maybe `const char *padstr = &table64[64]` instead? I think it makes the code clearer what it wants to point to.

Reply:
Done.